### PR TITLE
[SPARK-19626][YARN]Using the correct config to set credentials update time

### PR DESCRIPTION
**Pull request by yaooqinn**
## What changes were proposed in this pull request?

In https://github.com/apache/spark/pull/14065, we introduced a configurable credential manager for Spark running on YARN. Also two configs `spark.yarn.credentials.renewalTime` and `spark.yarn.credentials.updateTime` were added, one is for the credential renewer and the other updater. But now we just query `spark.yarn.credentials.renewalTime` by mistake during CREDENTIALS UPDATING, where should be actually `spark.yarn.credentials.updateTime` . 

This PR fixes this mistake.

## How was this patch tested?

existing test

cc @jerryshao @vanzin

### CODE REVIEW

File: resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/security/CredentialUpdater.scala [hunk 1]
```diff
@@ -55,14 +55,10 @@ private[spark] class CredentialUpdater(
...
-    val startTime = sparkConf.get(CREDENTIALS_RENEWAL_TIME)
-    val remainingTime = startTime - System.currentTimeMillis()
-    if (remainingTime <= 0) {
-      credentialUpdater.schedule(credentialUpdaterRunnable, 1, TimeUnit.MINUTES)
-    } else {
-      logInfo(s"Scheduling credentials refresh from HDFS in $remainingTime millis.")
-      credentialUpdater.schedule(credentialUpdaterRunnable, remainingTime, TimeUnit.MILLISECONDS)
-    }
+    val startTime = sparkConf.get(CREDENTIALS_UPDATE_TIME)
+    val remainingTime = Math.max(1, startTime - System.currentTimeMillis())
```
Comment (jerryshao) [inline resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/security/CredentialUpdater.scala]: The logic looks different compared to the above code, also `1` is minute nor millisecond.

**Review by jerryshao — CHANGES_REQUESTED** [review]
Thanks for the fix, only one comment about the changes.

File: resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/security/CredentialUpdater.scala (hunk 1 above)
Comment (yaooqinn) [inline resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/security/CredentialUpdater.scala]: OK, let me change it back

**Comment by yaooqinn** [comment]
cc again @jerryshao

**Comment by srowen** [comment]
There's no detail in the JIRA or PR. I can't tell whether you are solving the problem you set out to solve. Are you saying this is the entire problem, that the wrong config is being queried?

**Comment by jerryshao** [comment]
Agreed with @srowen , please describe the problem and fix both here in PR and in JIRA, also changing the title to be more meaningful. It would be better for others without the context to understand the issue fast.

**Comment by yaooqinn** [comment]
@srowen @jerryshao  Thanks for your comments. I have added some descriptions both in the JIRA and here, please check whether OK or not.

**Comment by yaooqinn** [comment]
ping @srowen, would you plz verify this patch again? thanks.

**Comment by srowen** [comment]
(I don't know enough about this to review)

**Comment by vanzin** [comment]
ok to test

**Comment by yaooqinn** [comment]
@vanzin Thanks, all tests passed

**Review by jerryshao — APPROVED** [review]

**Comment by vanzin** [comment]
Merging to master / 2.1.

**Pull request closed without merging**